js/.gitignore

```diff
@@ -8,3 +8,7 @@ docs
 .happypack
 .npmjs
 .eslintcache
+
+npm/*/src
+npm/*/dist
+npm/jsonrpc/index.json
```

Comment:
Shouldn't the LICENCE files be here now as well since it is added after the fact?